src/cargo/util/toml/mod.rs

```diff
@@ -1137,12 +1359,29 @@ impl TomlManifest {
 
         validate_package_name(package_name, "package name", "")?;
 
-        let pkgid = project.to_package_id(source_id)?;
+        let version = project.version.clone().resolve(
+            &features,
+            "version",
+            || get_ws(inheritable),
+            |i| i.version(),
```

Comment:
I feel like `get_ws` should either be renamed here or in `resolve`.  We are using a `get_ws` function for the `inherit` parameter and then we have a `get_ws` parameter.

Reply:
I agree with you on this, I think we should rename it in `resolve` but I will play around with what feels best